packages/patternfly-4/react-charts/src/components/ChartDonut/examples/ChartDonut.md

```diff
@@ -235,16 +237,15 @@ import { ChartDonut } from '@patternfly/react-charts';
       width={300}
     />
   </div>
-</div>
+)
 ```
 
-## Tips
+## Documentation
```

Comment:
We have "Docs" just below this section. These bullets are more tips or notes if you will

Reply:
I changed this to look a little better given that `<h2>` elements now become a part of the table of contents.

Reply:
The bullet list doesn't look like it's formatting properly?

<img width="719" alt="Screen Shot 2019-10-16 at 8 46 58 PM" src="https://user-images.githubusercontent.com/17481322/66969124-4fe87980-f056-11e9-9d39-6287b3e0583f.png">

Reply:
Correct, we currently aren't adding styles outside of what PF gives us for free to content other than `<h1>-<h6>`. This will change soon in a future update to `gatsby-theme-pattenfly-org`.